lib/logsource.h

```diff
@@ -33,7 +33,7 @@
 
 typedef struct _LogSourceOptions
 {
-  gint init_window_size;
+  gsize init_window_size;
```

Comment:
This is a `-1`-initialized value, let's use gssize.

(I know, `initial_window_size` is unsigned and we'd use half of the representable window sizes this way, but per-source windows should not be that high.)

Reply:
Thank you for the catch. Fixed

Reply:
Thanks! :)